### PR TITLE
feat: add conform webhook

### LINKED ISSUE #12: add conform to the project

### PR DESCRIPTION
**Pull request by smira**
Fixes #12

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — DISMISSED** [review]

File: internal/output/conform/template.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      types: {{ .Types }}
+      scopes: {{ .Scopes }}
+- type: license
+  spec:
+    skipPaths:
+      - .git/
+    includeSuffixes:
+      - .go
+    excludeSuffixes:
+      - .pb.go
```
Comment (andrewrynhard) [inline internal/output/conform/template.go:32]: We may need this configurable. I think there are a few projects that have something unique to them.

Comment (smira) [reply internal/output/conform/template.go:32]: yep, agreed, I went lazy route for now, but yeah, for sure. 

I was lazy to get full Go structs for it, so I ended up having something in between :)

**Pull request merged**